pkg/kubelet/kubelet.go

```diff
@@ -772,6 +772,14 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 		if err != nil {
 			return nil, err
 		}
+
```

Comment:
this should be a separate PR. i'd be tempted to group it with the Labels cloud cloud provider change, but worry that it might get nixed that way: this might be best as its own PR (and make a reference to the cloudprovider Labels PR so that people have some context)